firmware/src/app.c

```diff
@@ -31,6 +32,8 @@
 #include "transceiver.h"
 #include "usb_transport.h"
 
+const uint8_t OUR_UID[UID_LENGTH] = {0x7a, 0x70, 0xff, 0xff, 0xfe, 0};
```

Comment:
This should probably have a todo to move to a new range when Ja Rule gets a proper block allocated.